ibis/clickhouse/tests/test_client.py

```diff
@@ -47,7 +47,7 @@ def test_result_as_dataframe(con, alltypes):
     result = con.execute(expr)
 
     assert isinstance(result, pd.DataFrame)
-    assert list(result.columns) == ex_names
+    assert sorted(result.columns) == sorted(ex_names)
```

Comment:
Is it possible that this fails without sorting?

Reply:
It did previously, but I've fixed that later.

Reply:
Ok great!